mindsdb/integrations/handlers/groq_handler/README.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Groq
+sidebarTitle: Groq
+---
+
+This documentation describes the integration of MindsDB with [Groq](https://groq.com/), a cloud service that simplifies the way developers interact with cutting-edge LLMs through its API.
+
+## Prerequisites
+
+Before proceeding, ensure the following prerequisites are met:
+
+1. Install MindsDB locally via [Docker](https://docs.mindsdb.com/setup/self-hosted/docker) or [Docker Desktop](https://docs.mindsdb.com/setup/self-hosted/docker-desktop).
+2. To use Groq within MindsDB, install the required dependencies following [this instruction](https://docs.mindsdb.com/setup/self-hosted/docker#install-dependencies).
+3. Obtain the Groq API key required to deploy and use Groq models within MindsDB. Get the API key from [here](https://console.groq.com/keys).
+
+## Setup
+
+Create an AI engine from the [Groq handler](https://github.com/mindsdb/mindsdb/tree/main/mindsdb/integrations/handlers/groq_handler).
+
+```sql
+CREATE ML_ENGINE groq_engine
+FROM groq
+USING
+      groq_api_key = 'api-key-value'
+```
+
+Create a model using `groq_engine` as an engine.
+
+```sql
+CREATE MODEL groq_model
+PREDICT answer
+USING
+      engine = 'groq_engine',   -- engine name as created via CREATE ML_ENGINE
+      model_name = 'model-name',              -- choose one of available models
+      prompt_template = 'prompt-to-the-model' -- prompt message to be completed by the model
+```
+
+## Usage
+
+The following usage examples utilize `groq_engine` to create a model with the `CREATE MODEL` statement.
```

Comment:
Since this is inheriting from the OpenAI integration, I believe that all of the parameters that are supported by the OpenAI integration are supported here as well?
Can we please list them here? You can use the [README](https://github.com/mindsdb/mindsdb/blob/main/mindsdb/integrations/handlers/openai_handler/README.md) of the OpenAI integration as a reference.

Reply:
Should I edit that USAGE part to
```sql
CREATE MODEL groq_model
PREDICT answer
USING
      engine = 'groq_engine',   -- engine name as created via CREATE ML_ENGINE
      model_name = 'model-name',              -- choose one of available models
      prompt_template = 'prompt-to-the-model' -- prompt message to be completed by the model
      question_column = 'question',  -- optional, column name that stores user input
      context_column = 'context',  -- optional, column that stores context of the user input
      prompt_template = 'input your query here', -- optional, user provides instructions to the model here
      user_column = 'user_input', -- optional, stores user input
      assistant_column = 'conversation_context', -- optional, stores conversation context
      prompt = 'instruction to the model', -- optional stores instruction to the model
      max_tokens = 100, -- optional, token limit for answer
      temperature = 0.3, -- temp
```

Reply:
Yep, looks good.